void/selector.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+""" 
+void_selector 0.1
+
+Finds images in a given directory which contain a given point or polygon.
+
+Usage:
+  void_selector SEARCH_DIR [--point=POINT | --poly_pts=POLY_PTS] \
+[--verbosity=V]
+  void_selector -v | --version
+  void_selector -h | --help
+
+Options:
+  -p --point          Point which is checked
+  -l --poly_pts       List containing the polygon points which are checked
+  -v --version        Show program name and version number
+  -V --verbosity=V    Logging verbosity, 0 to 4 [default: 2]
+"""
+
+
+def point_within_area(point, area_ll, area_ur):
+    ra_p, dec_p = point
+    ra_ll, dec_ll = area_ll
+    ra_ur, dec_ur = area_ur
+    if ra_ll < ra_p < ra_ur and dec_ll < dec_p < dec_ur:
```

Comment:
This calculation needs to be done on the DB level, by PostGIS plugin. Can you open a separate branch for this file and remove it from this PR 🙏 , or stash it locally?